### ISSUE TITLE
Incomplete page load on iPads (companion app only)

### ISSUE THREAD

**Issue by fmonday** [home-assistant/frontend#9602]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

For at least a year, maybe more, I've had frequent (though unpredictable) partial page loads with gray zones and incomplete elements on a 2018 IPad Pro; this is best illustrated by looking at the pics which show the incomplete load and then the correct view. Moving the page at all with a finger causes the page to correct itself. This wasn't really much of an issue until my system finally matured to the point that I'm ready to mount it on the wall with a motion sensor turning it on when approached; now the incomplete load is an eyesore. 
The issue is not replicated using Safari or Chrome for iOS though the [known flickering issue](https://github.com/home-assistant/frontend/issues/8177) does happen on the mobile browsers. 

![C0BB633F-3371-4D8F-B946-F58BE4842783](https://user-images.githubusercontent.com/43834799/126913812-2df79ddc-5fc6-48a5-8640-0d1192ac4bcd.jpeg)

![6B85E8A9-62CF-4D7A-BDA1-78917B643308](https://user-images.githubusercontent.com/43834799/126913827-e823df5a-c355-4cca-9319-cb3f96798925.jpeg)

### Describe the behavior you expected

Pages load as intended. 

### Steps to reproduce the issue

1.  Work in a different app than the companion app. 
2.  Command-tab back to the HA companion app. Page loads in completely. 
3.  Jiggle screen w/ finger; page corrects. 
...

### What version of Home Assistant Core has the issue?

Core-2021.7.4, Python 3.9.4 in VENV

### In which browser are you experiencing the issue with?

2021.7(2021.202)

### Which operating system are you using to run this browser?

IOS 14.7

### Problem-relevant frontend configuration

```yaml
Page zoom 100%, pinch to zoom is off.
```

**Comment by bramkragten** [comment]
This is just about the background right? Please make that clear in the description and title.

**Comment by fmonday** [comment]
It's not about the background. As you can see in comparing those first few photos elements are missing including two custom button cards on the upper right, the knobs on the sliders, and also icons disappear from the menu bars including the hamburger menu, overflow menu and most of the sidebar. It does seem to be the same things that are missing when it happens.

**Comment by bramkragten** [comment]
You should start with removing all custom stuff, and see if it still happens then.

**Comment by fmonday** [comment]
So I followed the directions to completely delete HACS, /www/community, all custom components, everything related in .storage, all the required restarts etc. Which yielded a bunch of error cards so who knows. So I reinstalled HACS from scratch and started slowly adding back integrations/plugins and the issue immediately came back. Interestingly one of the main places the issue occurs is in the HACS frontend page so it occurred to me that HACS is the issue, but there's no way to test that. Honestly because it only happens on the companion app I think it's something about the browser wrapper
![94D5FE65-EED3-4F24-B19D-D1502A5B3712](https://user-images.githubusercontent.com/43834799/129983240-9f0bb1d5-1e0f-4617-9059-8af3f463c067.png)
